### PR TITLE
Fixed the offsets for Tiberian Sun transformable units

### LINKED ISSUE #5906: Transforms Offset ignored in TS
Tick tank and artillery always deploy north of the vehicle position, regardless of the Offset.

<bountysource-plugin>
---
Want to back this issue? **[Post a bounty on it!](https://www.bountysource.com/issues/3043662-transforms-offset-ignored-in-ts?utm_campaign=plugin&utm_content=tracker%2F36085&utm_medium=issues&utm_source=github)** We accept bounties via [Bountysource](https://www.bountysource.com/?utm_campaign=plugin&utm_content=tracker%2F36085&utm_medium=issues&utm_source=github).
</bountysource-plugin>

### PR DESCRIPTION
**Pull request by Mailaender**
Fixes #5906 though it seems already fixed as the issue is old. We just used incorrect offset values here as far as I see it. Those may have worked around isometric perspective problems in the past. I don't remember. Anyway, this tweaks them.

Also for some reason the lightning of the artillery voxel is quite different compared to http://cnc.wikia.com/wiki/Artillery_(Tiberian_Sun) and the sprite so I essentially did the opposite of https://github.com/Phrohdoh/oramod-ra2/pull/56/ to fix it.

### CODE REVIEW

File: mods/ts/rules/nod-vehicles.yaml [hunk 1]
```diff
@@ -133,10 +132,10 @@ ART2:
 		Range: 9c0
 	RenderSprites:
 	RenderVoxels:
+		NormalsPalette: ra2-normals
```
Comment (pchote) [inline mods/ts/rules/nod-vehicles.yaml]:
```
# Voxel   Limb          NormalsType
art2      DUMMY01ING019 TiberianSun
art2tur   A_TURRET B    TiberianSun
art2barl  DUMMY01@      TiberianSun
```

This doesn't use RA2-style normals (afaik that older engine didn't understand them).

Comment (Mailaender) [inline mods/ts/rules/nod-vehicles.yaml]: Suggestions on how else to fix this?

Comment (reaperrr) [inline mods/ts/rules/nod-vehicles.yaml]: Set ```LightAmbientColor: 0.4, 0.4, 0.4``` instead.

Comment (reaperrr) [inline mods/ts/rules/nod-vehicles.yaml]: And don't forget the deployed artillery (because of the barrel).

Comment (Mailaender) [inline mods/ts/rules/nod-vehicles.yaml]: Thanks. Updated.

**Comment by pchote** [comment]
There are still other polish issues with these units, but the changes in this pr work as advertised :+1:

**Comment by reaperrr** [comment]
:+1:

**Pull request merged**